AirCasting/Settings/BackendSettingsView.swift

```diff
@@ -62,8 +62,12 @@ struct BackendSettingsView: View {
             urlProvider.baseAppURL = url ?? URL(string: "http://aircasting.org/")!
             presentationMode.wrappedValue.dismiss()
             do {
-                userState.isLoggingOut = true
-                try logoutController.logout { userState.isLoggingOut = false }
+                userState.isShowingLoading = true
```

Comment:
@jankrzempek do you know why are we logging user out when changing backend url?

Reply:
Yes! So the thing is: when you decide to change the backend to the one you own or some other mysterious one, you need to log out to clear all of the things that you have there What I mean by that: all of the sessions and settings. The other, probably the most important thing is to log in once again (so first we need to log out) - this way we are sure that new logging in happened on the behalf of the new backend, and all of the sessions will be from there.